UIKItTemplate/Modules/Profile/Controller/ProfileViewController.swift

```diff
@@ -0,0 +1,42 @@
+// ProfileViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Class Controller
+class ProfileViewController: UIViewController {
+    enum Constants {
+        enum Text {
+            static let title = "Профиль"
+        }
+    }
+
+    // MARK: - Constants
```

Comment:
```suggestion
    // MARK: - Constants
```
это все пустое удаляем

Reply:
Удалил